src/snowflake/snowpark/_internal/utils.py

```diff
@@ -371,6 +379,22 @@ def create_statement_query_tag(skip_levels: int = 0) -> str:
     return "".join(stack[:-skip_levels] if skip_levels else stack)
 
 
+def create_or_update_statement_params_with_query_tag(
+    statement_params: Optional[Dict[str, Any]] = None,
+    exists_session_query_tag: Optional[str] = None,
+    skip_levels: int = 0,
+) -> Dict[str, Any]:
```

Comment:
In another spot you defined this as `Dict[str, str]`, please be consistent!

Reply:
updated, thanks for catching it!